lib/solveEquation/stepThrough.js

```diff
@@ -50,9 +51,23 @@ function stepThrough(leftNode, rightNode, comparator, debug=false) {
   const MAX_STEP_COUNT = 20;
   let iters = 0;
 
+  // Checks if there are roots in the original equation before we
```

Comment:
@aliang8 , @evykassirer says "I’m pretty sure you don’t have to make the second call, it’ll just catch next time it loops or looks for steps" -- does this makes sense to you? Can you address this?